README.markdown

```diff
@@ -79,6 +79,37 @@ Caveman2 is now available on [Quicklisp](https://www.quicklisp.org/beta/).
 ;   writing /path/to/myapp/templates/layout/default.tmpl
 ```
 
+### Start a server
+This is a example that assumes that the name of your application is "myapp".
+Before starting the server, you must first load your app.
+
+```common-lisp
+(ql:quickload :myapp)
+```
+
+Your application has functions named `start` and `stop` to start/stop your web application. 
+
+```common-lisp
+(myapp:start :port 8080)
+```
+
+As Caveman is based on Clack/Lack, you can choose which server to run on -- Hunchentoot, mod_lisp or FastCGI.
```

Comment:
Changing the list of example web servers.
This is an original text, but I'd like to change at this good chance.

```suggestion
As Caveman is based on Clack/Lack, you can choose which server to run on -- Hunchentoot, Woo or Wookie, etc.
```